docs/design/cincinnati.md

```diff
@@ -49,11 +49,20 @@ An example of this metadata document can be seen here:
 }
 ```
 
-A simple example of this directed acyclic graph can be seen in Figure 1. In this example, four versions were released (`1.0.0`, `1.1.0`, `1.2.0`, and `1.3.0`). After releasing `1.1.0`, it is discovered that there is a critical bug and this version is subsequently removed from the graph. Because of this, it will not be possible for clients who are currently running `1.0.0` to update to `1.2.0`.
+A simple example of this directed acyclic graph can be seen in Figure 1. In this example, four versions were released (`1.0.0`, `1.1.0`, `1.2.0`, and `1.3.0`).
 
 <figure align="center">
-  <img src="figures/cincinnati-1.svg" alt="Figure 1: A visualized DAG with three groups and a broken release" />
-  <figcaption>Figure 1: A visualized DAG with three groups and a broken release</figcaption>
+  <img src="figures/cincinnati-initial.svg" alt="Figure 1: A visualized DAG with four releases" />
+  <figcaption>Figure 2: A visualized DAG with four releases</figcaption>
+</figure>
+
+After releasing `1.1.0`, it is discovered that there is a critical bug and this version is subsequently flagged as degraded.
+Edges which lead into the degraded release are removed to prevent clients from upgrading to the degraded release.
+Because of this, it will not be possible for clients who are currently running `1.0.0` to update to `1.2.0` (FIXME: can we burn in a new edge from 1.0.0 direct to 1.2.0 by updating 1.2.0's metadata document to include a new `previous` entry for `1.0.0`?).
```

Comment:
This is possible today by adding specific labels to quay tags from/to which you want to add/remove edges: https://github.com/openshift/cincinnati/blob/bf130b71146bf1f335de64f97e4ff9c1c481b2d3/cincinnati/src/plugins/internal/edge_add_remove.rs#L37-L39

Reply:
> This is possible today by adding specific labels to quay tags...

This means we could remove `next` and just have folks manipulate `previous` to add/remove edges.  But I've left that alone for now, and just mentioned the late-edge-addition possibility in 974dc6e -> b9cea22.  This lets me remove the WIP tag from the PR too.

Reply:
True that we don't strictly need both directions for adding/removing edges but it might come in handy and it wasn't significantly more effort, so I'll keep the functionality in.